src/CodeGeneration.Roslyn/Logger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CodeGeneration.Roslyn
+{
+    public enum LogLevel
+    {
+        /// <summary>
+        /// High importance, appears in less verbose logs
+        /// </summary>
+        High = 0,
+
+        /// <summary>
+        /// Normal importance
+        /// </summary>
+        Normal = 1,
+
+        /// <summary>
+        /// Low importance, appears in more verbose logs
+        /// </summary>
+        Low = 2
+
```

Comment:
Can you add a trailing `,` after the last enum item?
Also move your blank line that is before the `}` to after it?